firebase-firestore/src/main/java/com/google/firebase/firestore/core/SyncEngine.java

```diff
@@ -21,6 +21,8 @@
 import androidx.annotation.VisibleForTesting;
 import com.google.android.gms.tasks.Task;
 import com.google.android.gms.tasks.TaskCompletionSource;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
```

Comment:
One of our goals is to eventually kick out Guava from our dependencies. Can you find another solution?

Reply:
Done.